internal/state/metrics.go

```diff
@@ -35,12 +43,29 @@ func PrometheusMetrics(namespace string, labelsAndValues ...string) *Metrics {
 			Help:      "Time between BeginBlock and EndBlock in ms.",
 			Buckets:   stdprometheus.LinearBuckets(1, 10, 10),
 		}, labels).With(labelsAndValues...),
+		ConsensusParamUpdates: prometheus.NewCounterFrom(stdprometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: MetricsSubsystem,
+			Name:      "consensus_param_updates",
+			Help: "The total number of times the application as updated the consensus " +
```

Comment:
This is fine, but I'd probably just let it be long.

Reply:
As in not wrap lines?

Reply:
Well, not "never" but it doesn't seem to help much in this case. Not a big deal though.